packages/core/src/fhir-to-cda/cda-templates/clinical-document.ts

```diff
@@ -0,0 +1,93 @@
+import { XMLBuilder } from "fast-xml-parser";
+
+// Constants for dynamic values
+const CONSTANTS = {
+  realmCode: "US",
+  typeIdExtension: "POCD_HD000040",
+  typeIdRoot: "2.16.840.1.113883.1.3",
+  templateIds: [
+    { root: "1.2.840.114350.1.72.1.51693" },
+    { root: "2.16.840.1.113883.10.20.22.1.1", extension: "2015-08-01" },
+    { root: "2.16.840.1.113883.10.20.22.1.9", extension: "2015-08-01" },
+  ],
+  assigningAuthorityName: "METRIPORT",
+  idRoot: "OUR-ORGANIZATION-ID",
+  code: {
+    code: "<NOTE-TYPE>",
+    codeSystem: "2.16.840.1.113883.6.1",
+    codeSystemName: "LOINC",
+    displayName: "<NOTE-NAME>",
+  },
+  title: "<NOTE-TITLE>",
+  effectiveTime: "<EFFECTIVE-TIME>", // TODO: Replace with current date. IMPORTANT
+  confidentialityCode: {
+    code: "N",
+    codeSystem: "2.16.840.1.113883.5.25",
+    displayName: "Normal",
+  },
+  languageCode: "en-US",
+  setId: {
+    assigningAuthorityName: "METRIPORT",
+    extension: "<OUR-EXTENSION>",
+    root: "<OUR-ID>",
+  },
+  versionNumber: "3",
+};
+
+// see https://build.fhir.org/ig/HL7/CDA-core-sd/StructureDefinition-ClinicalDocument.html
+export function constructClinicalDocumentXML(
+  recordTarget: unknown,
```

Comment:
using unknowns. I thought it would be cumbersome to define types also for all the jsons, but maybe its valuable to do?